src/d3a/setup/strategy_tests/smart_meter_setup.py

```diff
@@ -37,9 +37,9 @@ def get_setup(config):
                 grid_fee_percentage=0, grid_fee_constant=0,
             ),
             Area("House 2", [
-                Area("H2 Home Meter", strategy=HomeMeterStrategy(
+                Area("H2 Smart Meter", strategy=SmartMeterStrategy(
```

Comment:
An additional nesting of area for House2 could be avoided here as House 2 anyway have lumped energy reading of house via `H2 Smart Meter`

Reply:
Any response to this comment?